scripts/find_pulsar_in_obs.py

```diff
@@ -113,24 +113,26 @@ def get_psrcat_ra_dec(pulsar_list=None, max_dm=1000., include_dm=False):
     """
     import psrqpy
 
-    params = ['JNAME', 'RAJ', 'DECJ', 'DM']
-    query = psrqpy.QueryATNF(params=params, psrs=pulsar_list, loadfromdb=ATNF_LOC).pandas
+    #params = ['JNAME', 'RAJ', 'DECJ', 'DM']
+    if not query:
+        query = psrqpy.QueryATNF(params = ['PSRJ', 'RAJ', 'DECJ', 'DM'], psrs=pulsar_list, loadfromdb=ATNF_LOC).pandas
 
     pulsar_ra_dec = []
-    for row in query.itertuples():
+    for i, psr in enumerate(query["PSRJ"]):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused variable 'psr'](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=5199130)